### PR TITLE
Added instructions for Ulysses

### PR DESCRIPTION
**Pull request by defectivebit**
The metadata at the top will be updated later (dracula/ulysses#1).

### CODE REVIEW

File: src/content/ulysses/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	<p>Clone or <a href="https://github.com/dracula/ulysses.git/archive/master.zip">download</a> the repo, then open the Dracula.ultheme file.</p>
+
+	<h4>Activating theme</h4>
+
+	<ol>
+		<li>Open Ulysses' preferences <em>(Ulysses > Preferences...)</em></li>
+		<li>Click <em>Markup</em></li>
+		<li>Choose <em>Dracula</em> in the list of themes at the bottom.</li>
+	</ol>
+</div>
```
Comment (nesl247) [inline src/content/ulysses/index.html]: Please fix the ending newline.

File: src/content/ulysses/index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<div class="instructions">
+	<h3>Ulysses</h3>
+
+	<h4>Ulysses Style Exchange</h4>
+
+	<p>You can get Dracula from the <a href="http://styles.ulyssesapp.com/bundle/Dracula/56319778b634a4410207d126">Ulysses Style Exchange</a>.</p>
+
+	<h4>Install manually</h4>
+
+	<p>Clone or <a href="https://github.com/dracula/ulysses.git/archive/master.zip">download</a> the repo, then open the Dracula.ultheme file.</p>
```
Comment (nesl247) [inline src/content/ulysses/index.html]: Please see some of the other themes on how they handle this. We like to put the exact instructions on how to clone the repo.

**Comment by nesl247** [comment]
As much as I'd love to accept this, the icon would be duplicated which we don't want. We are in the process of powering the site via the READMEs, so if you can actually add the instructions there (See the Alfred theme), it would be appreciated.

Please change this PR to only include the screenshot. Thanks!

**Comment by nesl247** [comment]
As much as I'd love to accept this, the icon would be duplicated which we don't want. We are in the process of powering the site via the READMEs, so if you can actually add the instructions there (See the Alfred theme), it would be appreciated.

Please change this PR to only include a screenshot. Thanks!

**Comment by nesl247** [comment]
I want to apologize if things have been a bit confusing. We've been doing a lot of work to make it easier to contribute to dracula, and we might have gone back and forth a few times. Sorry about that.

**Comment by zenorocha** [comment]
Hey guys!

We still don't have all the infrastructure or icons we wanted but I decided to go ahead, merge this, make some changes, and put it live so other people can benefit from it.

https://draculatheme.com/ulysses/

Thank you so much for the time you both invested on this.

**Comment by defectivebit** [comment]
No problem.

When I have spare time, I don't mind contributing to the website effort. I'm wondering whether a design without the icons might better support the growing number of editors being added? I would think that once the scripts, etc. get set up it would make it much easier to add themes to the website.

**Pull request merged**